docs/src/charmil_logger.md

```diff
@@ -13,7 +13,7 @@ Errorln(args ...interface{})
 ```
 
 ## How to use
-When the factory is initialised, the logger is provided by default. To know about factory visit [here](charmil_factory.md)
+When the factory is initialized, the logger is provided by default. To know about factory visit [here](charmil_factory.md)

```

Comment:
awesome @jjkiely 
there were a lot of mistakes 😅 Thanks for correcting them!

Reply:
No problem 😊
I had to use a dictionary to figure out of locale or local was correct in this context 😂